### PR TITLE
NEUBURG-24: Remove integreat branding

### PR DESCRIPTION
**Pull request by michael-markl**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/app/containers/RouterFragment.js [hunk 1]
```diff
@@ -30,6 +34,9 @@ export class RouterFragment extends React.Component {
     return <Fragment forRoute='/'>
       {/* Routes */}
       <React.Fragment>
+        <Helmet>
+          <title>{cityConfig.title}</title>
```
Comment (steffenkleinle) [inline src/modules/app/containers/RouterFragment.js:38]: So we always have the same title? And I think in NEUBURG-17 you removed title from the cityConfig.

**Review by steffenkleinle — CHANGES_REQUESTED** [review]

File: src/modules/app/containers/RouterFragment.js (hunk 1 above)
Comment (michael-markl) [reply src/modules/app/containers/RouterFragment.js:38]: Yep, I wasn't expecting that I'd need the title anywhere :D will fix it when merging.
Same title in all pages is alright for me for now ^^ can be improved if needed

Comment (steffenkleinle) [reply src/modules/app/containers/RouterFragment.js:38]: I don't know if it's required, for me it's also okay if we leave it  like this.

**Review by steffenkleinle — APPROVED** [review]

File: www/index.ejs [hunk 2]
```diff
@@ -3,30 +3,16 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
-  <title><%= config.title %></title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
```
Comment (maxammann) [inline www/index.ejs:9]: Is there any favicon right now then? We have to ask neuburg probably about this.

File: www/index.ejs [hunk 3]
```diff
@@ -3,30 +3,16 @@
...
 
   <meta name="mobile-web-app-capable" content="yes">
-  <meta name="theme-color" content="#fbda16">
-  <meta name="msapplication-navbutton-color" content="#fbda16">
   <meta name="msapplication-starturl" content="/">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-  <link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
-  <link rel="apple-touch-icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (maxammann) [inline www/index.ejs:16]: Do we use this in the neuburg front-end or the integreat-webapp?

**Review by maxammann — APPROVED** [review]

File: www/index.ejs (hunk 2 above)
Comment (michael-markl) [reply www/index.ejs:9]: No currently not, but we've already asked about it.

File: www/index.ejs (hunk 3 above)
Comment (michael-markl) [reply www/index.ejs:16]: In neuburg-frontend only

**Comment by michael-markl** [comment]
Not sure why this was closed automatically... 🤔

**Pull request merged**